release/changes.md

```diff
@@ -0,0 +1 @@
+- [NEW] Run tests against AGP 8.4.0-alpha01
```

Comment:
The readme says `Supported Android Gradle Plugin versions: 7.0+, including alpha, beta, and rc versions` so we don't need to add this to the release notes since it doesn't change anything for the consumer.